pkgs/applications/video/kodi/addons/trakt-py/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildKodiAddon, fetchzip, addonUpdateScript, requests, six, arrow }:
+buildKodiAddon rec {
+  pname = "trakt-py";
+  namespace = "script.module.trakt";
+  version = "4.4.0";
```

```suggestion
  version = "4.4.0+matrix.1";
```

Looks like http://mirrors.kodi.tv/addons/matrix/addons.xml.gz has the `+matrix.1` version suffix.